app/build.gradle

```diff
@@ -6,6 +6,10 @@ android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
 
+    buildFeatures {
+        viewBinding = true
+    }
+
     defaultConfig {
         applicationId "com.example.pokemontcgapp"
```

Comment:
please use the package as com.globant. pokemontcgapp

Reply:
changed thanks